rmw_fastrtps_cpp/src/functions.cpp

```diff
@@ -672,6 +677,16 @@ rmw_node_t * rmw_create_node(const char * name, size_t domain_id)
   }
   memcpy(const_cast<char *>(node_handle->name), name, strlen(name) + 1);
 
+  node_handle->namespace_ =
+    static_cast<const char *>(malloc(sizeof(char) * strlen(namespace_) + 1));
```

Comment:
Actually, since the rest of the code around it uses `malloc`, I'm going to leave it for consistency. At some point we should audit memory allocation in Fast-RTPS.